logging-modules/flogger/pom.xml

```diff
@@ -17,11 +17,50 @@
             <artifactId>flogger</artifactId>
             <version>0.4</version>
         </dependency>
+
         <dependency>
             <groupId>com.google.flogger</groupId>
             <artifactId>flogger-system-backend</artifactId>
             <version>0.4</version>
             <scope>runtime</scope>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.flogger</groupId>
+            <artifactId>flogger-slf4j-backend</artifactId>
+            <version>0.4</version>
+        </dependency>
+
+<!--        <dependency>-->
```

Comment:
Could you please let me know why this part is left in code? If it is not necessary, let's remove it. Make sure also to run integration tests and check if they all pass.

Reply:
We added a section in the article about using other backends such as Slf4j and Log4j and using their existing configuration files. The dependencies that are commended out are for Log4j. This is so that the reader of the article can easily reference these dependencies. We could uncomment them, but still, I believe they are necessary for the reader.